backend/utils/message_utils.py

```diff
@@ -1,4 +1,4 @@
-from typing import Any, Optional, List, Dict
+from typing import Any, Dict, List, Optional
```

Comment:
We no longer need to import List and Dict from Python 3.9.
https://peps.python.org/pep-0585/
list and dict can be used. They are built in

Reply:
@devvspaces